src/redux/CompaniesList.js

```diff
@@ -0,0 +1,106 @@
+// import { createAsyncThunk } from '@reduxjs/toolkit';
+
+// const FETCH_COMPANIES_LIST_REQUEST = 'FETCH_COMPANIES_LIST_REQUEST';
+// const FETCH_COMPANIES_LIST_SUCCESS = 'FETCH_COMPANIES_LIST_SUCCESS';
+// const FETCH_COMPANIES_LIST_FAILURE = 'FETCH_COMPANIES_LIST_FAILURE';
+
+// const fetchCompaniesList = createAsyncThunk(
+//   'companies/fetchCompaniesList',
+//   async () => {
+//     const url = 'https://mattermarkraygorodskijv1.p.rapidapi.com/getCompaniesList';
+//     const options = {
+//       method: 'POST',
+//       headers: {
+//         'X-RapidAPI-Key': '032b57fca6mshe34316a66b67d92p1e8431jsn944052e12892',
+//         'X-RapidAPI-Host': 'MattermarkraygorodskijV1.p.rapidapi.com',
+//       },
+//     };
+
+//     try {
+//       const response = await fetch(url, options);
+//       const result = await response.text();
+//       console.log(result);
+//       return result;
+//     } catch (error) {
+//       console.error(error);
+//       throw error;
+//     }
+//   },
+// );
+
+// // Reducer function
+// const companiesReducer = (state = initialState, action) => {
+//   switch (action.type) {
+//     case FETCH_COMPANIES_LIST_REQUEST:
+//       return {
+//         ...state,
+//       };
+//     case FETCH_COMPANIES_LIST_SUCCESS:
+//       return {
+//         ...state,
+//         companiesList: action.payload,
+//         error: null,
+//       };
+//     case FETCH_COMPANIES_LIST_FAILURE:
+//       return {
+//         ...state,
+//         companiesList: null,
+//         error: action.payload,
+//       };
+//     default:
+//       return state;
+//   }
+// };
+
+// export { fetchCompaniesList };
+// export default companiesReducer;
```

Comment:
- [ ] For best practices and clean code, kindly ensure that you remove all the commented and non used code lines from your code base